src/luarocks/cmd/config.lua

```diff
@@ -218,44 +231,39 @@ local function write_entries(keys, scope, do_unset)
    end
 end
 
-local function check_scope(flags)
-   local scope = flags["scope"]
-                 or (flags["local"] and "user")
-                 or (flags["project-tree"] and "project")
-                 or (cfg.local_by_default and "user")
-                 or "system"
-   if scope ~= "system" and scope ~= "user" and scope ~= "project" then
-      return nil, "Valid values for scope are: system, user, project"
-   end
-
-   return scope
+local function get_scope(flags)
```

Comment:
rename the argument here to `args` to match?

Reply:
changed a bunch of occurrences of `flags` in `cmd.lua` too.